definitions/Uno.def

```diff
@@ -1,58 +1,50 @@
 Bootstrap: docker
-From: tensorflow/tensorflow:2.11.0-gpu
+From: python:3.7
```

Comment:
Is UNO not using any GPUs? I don't see any GPU support in this definition file.

Reply:
Oh, yes, thanks. I pip install tensor flow and just changed it to be the gpu supported version. The base image is from python because of python version dependency using the IMPROVE lib.